VOKMockUrlProtocol.m

```diff
@@ -111,6 +118,16 @@ - (NSArray *)resourceNames
         [resourceName appendFormat:queryFormat, self.request.URL.query];
     }
     
+    if (shouldEncodeAuthHeader) {
+        NSDictionary *headerDict = self.request.allHTTPHeaderFields;
+        NSString *authHeader = headerDict[kHTTPHeaderFieldAuthorization];
+        if (authHeader) {
+            NSString *encodedHeader = [authHeader stringByAddingPercentEscapesUsingEncoding:NSUTF8StringEncoding];
+            NSString *authString = [NSString stringWithFormat:@"&authorization=%@", encodedHeader];
+            [resourceName appendString:authString];
```

Comment:
@vokal-isaac Pretty sure there's more I should be doing here in terms of `sha256`'ing this to work better with the bEncoding stuff, but I'm not 100% clear on what I need to do here to make it work. Any direction would be most appreciated.